### PR TITLE
support python 2 and 3

### PR DESCRIPTION
**Pull request by nickpell**
@emord @millerdev cc: @snopoke

### CODE REVIEW

File: fakecouch.py [hunk 1]
```diff
@@ -142,7 +144,7 @@ def _transform_view_results(self, view_results):
 
     def _param_key(self, params):
         params = encode_params(params)
-        return urlencode(sorted(params.items(), key=lambda p: p[0]))
+        return urlencode(sorted(list(params.items()), key=lambda p: p[0]))
```
Comment (millerdev) [inline fakecouch.py]: nit: `list()` not necessary

**Comment by emord** [comment]
add py3? https://github.com/dimagi/fakecouch/blob/master/.travis.yml#L3

File: setup.py [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 try:
     from setuptools import setup
 except ImportError:
     from distutils.core import setup
 
 setup(
     name='fakecouch',
-    version='0.0.10',
+    version='0.1.0',
     description='Fake implementation of CouchDBKit api for testing purposes',
```
Comment (millerdev) [inline setup.py]: I can't find a definitive answer, but I think the way to indicate that this package supports both Python 2 and 3 is to add classifiers:
```py
    classifiers=[
         "Programming Language :: Python :: 2",
         "Programming Language :: Python :: 3",
    ],
```
If some point releases are supported but not others that can be indicated too:
```py
         "Programming Language :: Python :: 2.7",
```

**Comment by nickpell** [comment]
tests are blocking on updating couchdbkit

**Pull request merged**